docs/conf.py

```diff
@@ -100,7 +105,12 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
```

Comment:
This patterns also effect to html_static_path and html_extra_path
should read
These patterns also affect html_static_path and html_extra_path

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/454/commits/1ce506606220a371d145c7e4e0b3f1a271b71103